api/v1alpha1/serviceintentions_webhook_test.go

```diff
@@ -3,18 +3,20 @@ package v1alpha1
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"testing"
 
 	logrtest "github.com/go-logr/logr/testing"
 	"github.com/stretchr/testify/require"
+	"gomodules.xyz/jsonpatch/v2"
 	"k8s.io/api/admission/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-func TestValidateServiceIntentions_Create(t *testing.T) {
+func TestHandle_ServiceIntentions_Create(t *testing.T) {
```

Comment:
These tests actually call `Handle()` unlike the other webhook tests that do call `Validate()`